bundles/http/Citrix/Workday-Demo/script.js

```diff
@@ -0,0 +1,192 @@
+/*
+Name: Workday Demo HTTP integration, using generated dummy data
+Created by: Andrey Kornilov
+Date: 18.7.2021
+*/
+
+const _ = library.load('lodash')
+
+const names = ["Aline Gomes", "Billy Taylor", "Danielle Ledford", "Diana Sarkozy", "Erberto Tirado"]
+const descriptions = ["Lyft","Uber","Dinner","Hotel","Flight Tickets","Rental"]
+
+//Function, that generates data for the table, using random values, both for incremental and full sync
+async function getExpenses (dataStore,context,fullSync) {
+  let iterations;
+  let dateRandom;
+  const min = 100
+  const max = 2000
+//Switch statement, that regulates behavior of loop, depending on sync type
+  switch (fullSync) {
+    case true:
+      context.key1 = 1;
+      iterations = context.key1 + 15
```

Comment:
magic constants 15 and 3, why are these values chosen? refactor them to named constants so that it's purpose is obvious from their names or add line comment for them